web/src/topic_list_data.ts

```diff
@@ -108,7 +140,19 @@ function choose_topics(stream_id, topic_names, zoomed, topic_choice_state) {
     }
 }
 
-export function get_list_info(stream_id, zoomed, search_term) {
+type TopicListInfo = {
+    items: TopicInfo[];
+    num_possible_topics: number;
+    more_topics_unreads: number;
+    more_topics_have_unread_mention_messages: boolean;
+    more_topics_unread_count_muted: boolean;
+};
+
+export function get_list_info(
+    stream_id: number,
+    zoomed: boolean,
+    search_term: string,
+): TopicListInfo {
     const topic_choice_state = {
```

Comment:
Should this be marked as a `TopicChoiceState` here?

Reply:
Similarly for the `const topic_info` line in `choose_topics`, should we explicitly declare it as a `TopicInfo`?

Reply:
@evykassirer I have a slight preference for doing this if it's not much bother.

Reply:
Yeah sure! Done and pushed